xformers/components/attention/csrc/cuda/mem_eff_attention/kernel_forward.h

```diff
@@ -803,6 +805,15 @@ struct AttentionKernel {
   }
 };
 
+template <typename AK>
+__global__ void __launch_bounds__(AK::kNumThreads, AK::kMinBlocksPerSm)
+    attention_kernel_batched_impl(typename AK::Params p) {
```

Comment:
Is this a new function?

Reply:
In our code, we call `attention_kernel_batched`, which is implemented in a different file (because we don't want to compile too many things by mistake).
For cutlass, I'll call `attention_kernel_batched_impl`, which is always available.